molecule/default/molecule.yml

```
@@ -79,6 +79,14 @@ platforms:
       - /sys/fs/cgroup:/sys/fs/cgroup:rw
     cgroupns_mode: host
     privileged: true
+  - name: fedora-41  # EOL 2025-11-19
+    image: fedora:41
```

I believe that the `dokken/fedora-41` container image is needed instead of the `fedora-41` container image.

The `fedora` container image is provided by the Fedora project and probably does not include all the things that are necessary for use with Molecule.
```suggestion
    image: dokken/fedora-41:latest
```